src/DependencyInjection/Configuration.php

```diff
@@ -91,6 +91,8 @@ public function getConfigTreeBuilder(): TreeBuilder
                             ->end()
                         ->end()
                     ->end()
+                ->scalarNode('customFieldsPath')
```

Comment:
Wo wird die config wieder ausgelesen?

Reply:
Aktuell wird die nur in den Container injected und soll später dann ausgelesen werden um daraus die Manifest zu erstellen.